fuel/src/main/kotlin/com/github/kittinunf/fuel/core/requests/SuspendingRequest.kt

```diff
@@ -0,0 +1,28 @@
+package com.github.kittinunf.fuel.core.requests
+
+
+import com.github.kittinunf.fuel.core.FuelError
+import com.github.kittinunf.fuel.core.Request
+import com.github.kittinunf.fuel.core.Response
+import com.github.kittinunf.result.Result
+import com.github.kittinunf.result.mapError
+import java.io.InterruptedIOException
+
+class SuspendingRequest(internal val request: Request) {
```

Comment:
🤔 Why `internal`?

Reply:
I have made it private as access is not currently required outside the class